packages/mobile/src/screens/settings-screen/SettingsScreen.tsx

```diff
@@ -103,6 +115,17 @@ export const SettingsScreen = () => {
               {messages.notificationsDescription}
             </SettingsRowDescription>
           </SettingsRow>
+          <SettingsRow onPress={handlePressCommentSettings}>
```

Comment:
I think you want to move the `isCommentsEnabled` here? Not sure how this looks but I would imagine we remove the whole row if it's not enabled